test/functional/output_at_test.rb

```diff
@@ -69,11 +69,11 @@ class OutputAtTest < Whenever::TestCase
       set :job_template, nil
       set :path, '/your/path'
       every "mon,wed,fri", :at => '5:02am, 3:02pm' do
-        runner "blahblah"
+        runner "Worker.perform_async(1.day.ago)"
       end
     file
 
-    assert_match %(2 5,15 * * 1,3,5 cd /your/path && script/runner -e production 'blahblah'), output
+    assert_match %(2 5,15 * * 1,3,5 cd /your/path && script/runner -e production 'Worker.perform_async(1.day.ago)'), output
```

Comment:
@javan This spec pretty much explains the purpose of the PR.
In previous code, when we called `Whenever::NumericSeconds.process_string(schedule)` this spec would fail because the generated cron would be:

```
2 5,15 * * 1,3,5 cd /your/path && script/runner -e production 'Worker.perform_async(86400.ago)'
```

which is not something a user expects and also fails